samples/client/petstore/swift5/validation/PetstoreClient/Classes/OpenAPIs/APIs.swift

```diff
@@ -31,8 +23,6 @@ open class RequestBuilder<T> {
     public let requiresAuthentication: Bool
 
     /// Optional block to obtain a reference to the request's progress instance when available.
-    /// With the URLSession http client the request's progress only works on iOS 11.0, macOS 10.13, macCatalyst 13.0, tvOS 11.0, watchOS 4.0.
-    /// If you need to get the request's progress in older OS versions, please use Alamofire http client.
```

Comment:
I don't know why these changes are only in my branch, they appeared when I generated samples